gms/impl/src/test/resources/corpUserESSearchRequest.json

```diff
@@ -22,18 +22,17 @@
                     "fuzzy_transpositions" : true,
                     "lenient" : false,
                     "zero_terms_query" : "NONE",
+                    "auto_generate_synonyms_phrase_query": true,
```

Comment:
Similar question: what is the default value for this?

Reply:
> (Optional, Boolean) If true, match phrase queries are automatically created for multi-term synonyms. Defaults to true. See Synonyms for an example.

I don't know why this was added, I copied it from Jyoti's PR. I can remove it and see what happens?

Reply:
Ah, this is in test data. Yes, I'm guessing the ES java library makes some defaults explicit.